### DISCUSSION TITLE
Remember playback speed / option for default speed multiplier

### DISCUSSION THREAD

**Discussion by GameTec-live** [qnblackcat/uYouPlus#1251] [Ideas]
Currently the app, when closed and reopened doesnt remember the playback speed set. Id like the app to either remember the speed or have a setting for a new default playback speed (1x, 2x, etc...) which the app then automatically sets at startup.

**Comment by huntingwall8910** [comment]
I believe there is a default playback setting in the uYou settings![Uploading 275082871...]

**Reply by GameTec-live** [reply]
> I believe there is a default playback setting in the uYou settings![Uploading 275082871...]

I couldnt find one (and your image doesnt load;) )...
